src/pages/home/home.ts

```diff
@@ -239,6 +243,7 @@ export class HomePage {
     this.user$.subscribe(async user => {
```

Comment:
Since this subscription is happening inside of the ionViewWillEnter it's causing a flash everytime you enter settings. Simple fix is to move the subscription to the constructor.

Reply:
to be clear its just this block `this.user$.subscribe(async user => {
      if (user) {
        this.bitPayIdUserInfo = user;
        this.accountInitials = this.getBitPayIdInitials(user);
        this.changeRef.detectChanges();
      }
    });`

Reply:
Fixed! Thanks!